src/qibolab/platforms/platform.py

```diff
@@ -32,16 +36,29 @@ def disconnect(self):
         self.design.disconnect()
         self.is_connected = False
 
-    def execute_pulse_sequence(self, sequence, nshots=1024, relaxation_time=None, raw_adc=False):
-        if relaxation_time is None:
-            relaxation_time = self.relaxation_time
-        return self.design.play(self.qubits, sequence, nshots=nshots, relaxation_time=relaxation_time, raw_adc=raw_adc)
+    def execute_pulse_sequence(self, sequence, **kwargs):
```

Comment:
Add a docstring, please.

I know it was not present even before this PR, but since you're changing it, you should know what it does and how.
A brief one is fine, even just one line (especially if you can type the parameters, since native type hints are more useful than types in the docstrings, and they are providing the same level of documentation).